data/styles/clear/style-clear/symbols/anchor-m.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" version="1.1" viewBox="0 -960 20 20" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this `viewBox="0 -960 20 20"` looks strange, usually its ` viewBox="0 0 24 24"`

Reply:
fixed :+1: